Dockerfile

```diff
@@ -91,6 +95,13 @@ RUN apt-get -qq -y update && \
         tzdata \
         python3 \
         net-tools \
+        build-essential \
+        libssl-dev \
+    	# must use libssl1.1 when install mongodb on Ubuntu 22.04 \
+        libssl1.1 \
+        libffi-dev \
+        libsqlite3-dev \
+        liblzma-dev \
```

Comment:
I believe some of these packages are needed when we need to build a custom Python version, except for `libssl1.1` for mongodb, so we should move them to `RUN if [ -n "$PYTHON_VERSION" ]; ` block? This helps to reduce the image size when we just use the default Python from ubuntu.